src/AGCertificateVerifier.cpp

```diff
@@ -196,10 +192,13 @@ AGVerifyResult AGCertificateVerifier::verifyDNSName(const std::string &dnsName,
  * are allowed to use SHA1).
  *
  * @param store CA store
+ * @param dnsName Host name
  * @param certChain Certificate chain
+ * @param basicCheckOnly Don't perforn sha1 deprecation and HPKP checks
  * @return Verify result
  */
-AGVerifyResult AGCertificateVerifier::verifyChain(X509_STORE *store, STACK_OF(X509) *certChain) {
+AGVerifyResult AGCertificateVerifier::verifyChain(X509_STORE *store, STACK_OF(X509) *certChain,
+                                                  const std::string &dnsName, bool basicCheckOnly) {
     // Initialize cert store context
```

Comment:
Minor nitpick -- doxygen doc parameter order is different from actual parameter order.